kong/pdk/request.lua

```diff
@@ -841,6 +842,53 @@ local function new(self)
     return ngx.ctx.KONG_PROCESSING_START or (req.start_time() * 1000)
   end
 
+  local EMPTY = {}
+  local named_captures = {}
+  local unnamed_captures = {}
+  local wrapped_captures = {
```

Comment:
These tables is re-used when we call `capture_wrap`, Is it safe enough in concurrency?

Reply:
There's no yielding until the encoding finishes. But just in case I'm not going to reuse the table(it doesn't worth the risk).